tests/backbeat/Using.md

```diff
@@ -9,18 +9,29 @@
    127.0.0.1 zenko.local
    ```
 
-2. Create an account using Orbit.
-3. Export the access key and secret key of that account (for example, in
+2. Find the Zenko service endpoint for S3 calls to Cloudserver by running:
+    ```
+    kubectl get services
+    ```
+    Look for the service named `zenko-cloudserver`, the value under `CLUSTER-IP`
```

Comment:
Instead of the cluster IP it would be preferable to simply use the value under `NAME` which resolves in the DNS to the `CLUSTER-IP` but at least with the `NAME` we'll always resolve to the correct location (in case the IP ever changes).